src/views/htmlcontent/src/js/components/app.component.ts

```diff
@@ -310,72 +310,45 @@ export class AppComponent implements OnInit, AfterViewChecked {
-                    hasError: batch.hasError,
-                    selection: batch.selection,
-                    startTime: new Date(batch.executionStart).toLocaleTimeString(),
-                    endTime: new Date(batch.executionEnd).toLocaleTimeString()
-                };
-                if (batch.hasError) {
-                    self._messageActive = true;
-                }
-                for (let message of batch.messages) {
-                    let date = new Date(message.time);
-                    let timeString = date.toLocaleTimeString();
-                    messages.messages.push({time: timeString, message: message.message});
-                }
-                self.messages.push(messages);
-                self.messagesAdded = true;
-                for (let resultId = 0; resultId < batch.resultSetSummaries.length; resultId++) {
-                    let result = batch.resultSetSummaries[resultId];
-                    let totalRows = result.rowCount;
-                    let columnData = result.columnInfo;
-                    let dataSet: IGridDataSet = {
-                            dataRows: undefined,
-                            columnDefinitions: undefined,
-                            totalRows: undefined,
-                            resized: undefined,
-                            batchId: batch.id,
-                            resultId: result.id,
-                            maxHeight: undefined,
-                            minHeight: undefined
-                        };
-                    let columnDefinitions = [];
-
-                    for (let i = 0; i < columnData.length; i++) {
-                        // Fix column name for showplan queries
-                        let columnName = (columnData[i].columnName === 'Microsoft SQL Server 2005 XML Showplan') ?
-                                                    'XML Showplan' : columnData[i].columnName;
-                        if (columnData[i].isXml || columnData[i].isJson) {
-                            let linkType = columnData[i].isXml ? 'xml' : 'json';
-                            columnDefinitions.push({
-                                id: i.toString(),
-                                name: columnName,
-                                type: self.stringToFieldType('string'),
-                                formatter: self.hyperLinkFormatter,
-                                asyncPostRender: self.linkHandler(linkType)
-                            });
-                        } else {
-                            columnDefinitions.push({
-                                id: i.toString(),
-                                name: columnName,
-                                type: self.stringToFieldType('string'),
-                                formatter: self.textFormatter
-                            });
-                        }
+            switch (event.type) {
+                case 'complete':
+                    self.complete = true;
```

Comment:
I was going to fix this, but might as well since you are editing this area; in this case you should also set self.messagesAdded since one of the side effects of setting complete to true is another message is added to the message pane.